.github/workflows/publish.yml

```diff
@@ -0,0 +1,51 @@
+name: Publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🔒​ Check actor permission level
+        uses: skjnldsv/check-actor-permission@v2
+        with:
+          require: admin
```

Comment:
@skjnldsv Would this be enough as this is in line with https://github.com/nextcloud/nextcloud-vue/blob/master/.github/workflows/npm-publish.yml#L16-L21 or would we still need a separate repo on Nextcloud-releases for each package? (ref https://github.com/nextcloud/standards/issues/2#issuecomment-1102796417)

Reply:
Should be enough imho